packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -202,48 +227,36 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
             minAttestations,
             'necessary attestations',
           );
+
           if (found.length < minAttestations) {
             return;
           }
 
-          let lastEvidence;
-          for (const store of found) {
-            const next = store.get(txHash);
-            if (lastEvidence && !keyEQ(lastEvidence, next)) {
-              // Ignore conflicting evidence, but treat it as an error
-              // because it should never happen and needs to be prevented
-              // from happening again.
-              trace(
-                '🚨 conflicting evidence for',
-                txHash,
-                ':',
-                lastEvidence,
-                '!=',
-                next,
-              );
-              Fail`conflicting evidence for ${quote(txHash)}`;
-            }
-            lastEvidence = next;
-          }
-
           const riskStores = [...risks.values()].filter(store =>
             store.has(txHash),
           );
-          // take the union of risks identified from all operators
-          const risksIdentified = allRisksIdentified(riskStores, txHash);
 
-          // sufficient agreement, so remove from pending risks, then publish
-          for (const store of found) {
-            store.delete(txHash);
+          // Publish at the threshold of agreement
+          if (found.length === minAttestations) {
+            // take the union of risks identified from all operators
+            const risksIdentified = allRisksIdentified(riskStores, txHash);
+            trace('publishing evidence', evidence, risksIdentified);
+            publisher.publish({
+              evidence,
+              risk: { risksIdentified },
+            });
+            return;
           }
-          for (const store of riskStores) {
-            store.delete(txHash);
+
+          if (found.length === pending.getSize()) {
```

Comment:
before merging I'll add a test for if operator count changes in flight.
```suggestion
          if (found.length >= pending.getSize()) {
```